src/pages/verantwoording.tsx

```diff
@@ -26,9 +26,7 @@ interface StaticProps {
 
 export async function getStaticProps(): Promise<StaticProps> {
   const text = require('../locale/index').default;
-  const serializedContent = text.verantwoording.cijfers.map(function (
-    item: ICijfer
-  ) {
+  const serializedContent = text.verantwoording.cijfers.map((item: ICijfer) => {
     return { ...item, verantwoording: MDToHTMLString(item.verantwoording) };
   });
 
```

Comment:
nitpick; You don't need the return statement with a one-line lambda function.

```suggestion
  const serializedContent = text.verantwoording.cijfers.map((item: ICijfer) => ({ ...item, verantwoording: MDToHTMLString(item.verantwoording)});
```

Reply:
Also the type annotation `ICijfer` should be inferred by the compiler already